pkgs/development/python-modules/temporalio/default.nix

```diff
@@ -0,0 +1,67 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+  cargo,
+  rustPlatform,
+  rustc,
+  setuptools-rust,
+  poetry-core,
+  pythonOlder,
+  protobuf,
+  buildPackages,
+  types-protobuf,
+  typing-extensions,
+}:
+
+buildPythonPackage rec {
+  pname = "temporalio";
+  version = "1.10.0";
+  pyproject = true;
+
+  disabled = pythonOlder "3.9";
+
+  src = fetchFromGitHub {
+    owner = "temporalio";
+    repo = "sdk-python";
+    rev = "refs/tags/${version}";
+    fetchSubmodules = true;
+    hash = "sha256-0y89P3+6n1Bi70uGQPCazPMiCexzFSFNu/yzwxAXNkI=";
+  };
+
+  build-system = [ poetry-core ];
+
+  cargoDeps = rustPlatform.fetchCargoVendor {
+    inherit pname version src;
+    sourceRoot = "${src.name}/${cargoRoot}";
+    hash = "sha256-5UB8CjnkO7GF412ZC6AHq6MQFfHuI8mEYXNslrmYisY=";
+  };
+
+  cargoRoot = "temporalio/bridge";
+
+  nativeBuildInputs = [
+    cargo
+    rustPlatform.cargoSetupHook
+    rustc
+    setuptools-rust
+    protobuf
+  ];
+  preBuild = ''
+    export PROTOC=${buildPackages.protobuf}/bin/protoc
+  '';
+
+  dependencies = [
+    protobuf
+    types-protobuf
+    typing-extensions
+  ];
+  pythonImportsCheck = [ "temporalio" ];
+
```

Comment:
add pytestCheckHook to nativeCheckInputs

Reply:
This causes the tests to fail (because the binary artifact is a part of the test). Can I skip it?

Reply:
If you skip just that test that would be preferred